contracts/script/GenerateAlloc.s.sol

```diff
@@ -217,25 +197,34 @@ contract GenerateAlloc is Script {
         // We deploy this with Create3 because we can't set storage variables in constructor with vm.etch
 
         uint256 minDelay = getTimelockMinDelay();
-        (address[] memory proposers, address[] memory executors, address canceller) = getTimelockControllers();
+        address[] memory proposers = new address[](1);
+        proposers[0] = protocolAdmin;
+        address[] memory executors = new address[](1);
+        executors[0] = timelockExecutor;
```

Comment:
Is it setting it to anyone can execute?

Reply:
It allows having address(0) in env files so it's public
https://github.com/piplabs/story/blob/0ac4627edd4d410910a926ca02763e42679aea15/contracts/script/GenerateAlloc.s.sol#L132